### PR TITLE
Add overrider description for overridePolicy API

### PR DESCRIPTION
**Pull request by Poor12**
Signed-off-by: Poor12 <shentiecheng@huawei.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Add overrider description for overridePolicy API

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
None

### CODE REVIEW

**Review by XiShanYongYe-Chang — COMMENTED** [review]
It looks like you need to run the script `hack/update-codegen.sh`.

File: pkg/apis/policy/v1alpha1/override_types.go [hunk 1]
```diff
@@ -81,6 +81,8 @@ type RuleWithCluster struct {
 //
 // If more than one alternatives exist, they will be applied with following order:
```
Comment (RainbowMango) [inline pkg/apis/policy/v1alpha1/override_types.go:82]: Here is a typo, please help to fix it.

```diff
// If more than one alternative exists, they will be applied in the following order:
```

**Comment by RainbowMango** [comment]
> It looks like you need to run the script hack/update-codegen.sh.

```
$ make update
```

**Review by XiShanYongYe-Chang — COMMENTED** [review]
/lgtm

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**